frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss

```
@@ -1,10 +1,17 @@
-.odc-empty-state {
-  padding: 25px;
-  background: var(--pf-global--Color--light-200);
-  box-shadow: var(--pf-global--BoxShadow--inset);
+.odc-empty-section {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
 
-  &__card {
-    height: 100%;
-    text-align: center;
+  .odc-empty-state {
```

```suggestion
  &__content {
```
Then move `&__card` so it's nested below the block.